tests/conftest.py

```diff
@@ -16,4 +24,20 @@ def app():
 def client(app):
     """A test client for the app."""
     app.testing = True
-    return app.test_client()
\ No newline at end of file
+    return app.test_client()
+
+
+@pytest.fixture(autouse=False)
```

Comment:
I think `autouse=False` is the default and can be omitted.

Reply:
Done!